### DISCUSSION TITLE
I hate the new feed, Github isn't Facebook

### DISCUSSION THREAD

**Discussion by gbonacini** [community/community#66192] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I hate the new feed, I prefer the old version with stars and news about people I follow. I don't like to spammed here too. And anyway, selecting 'Send a Feedback' , I find myself in this "chat like" nightmare so apparently here even to send a feedback is a pain !

**Comment by hamidb80** [comment]
Yeah me too.

 Microsoft please respect developers ( can you ? ), don't make it another LinkedIn


But I'm really considering to move to code berg

**Comment by koppor** [comment]
Workaround: Use https://github.com/Gerrit0/old-github-feed
